docs/modules/development/pages/build-from-source.adoc

```diff
@@ -0,0 +1,141 @@
+[[build-from-source]]
+= Build from source
+
+We highly recommend running a *nix system to build OpenNMS from source code.
+You do not need to run it physically; a virtual machine is sufficient.
+Our community uses the following environments to develop our software:
+
+* link:https://en.wikipedia.org/wiki/List_of_Linux_distributions#Debian-based[Debian-based] operating system
+* link:https://en.wikipedia.org/wiki/List_of_Linux_distributions#RPM-based[RPM-based] operating system
+* Mac OS X
+
+This section describes how to build OpenNMS {page-component-title} with an Ubuntu Desktop system.
+Follow these instructions to accomplish the following:
+
+* Download the source code and create a build environment
+* A compiled and assembled version from the latest develop branch
+* An OpenNMS {page-component-title} core server instance running locally from a development directory
+
+You should be familiar with deploying OpenNMS {page-component-title}, especially setting up package repositories and configuring and initializing the PostgreSQL database.
+Basic knowledge of git is highly recommended.
+
+== Before you begin
+
+We publish and maintain the project on GitHub.
+Make sure you have access to GitHub to fetch the source with SSH or HTTPS.
+Once you have built the software, you need to install packages.
+Ensure your user account has administrative permissions with `sudo`.
```

Comment:
@Bonrob2 moved these two issues out of the install packages list and added them here. What do you think?

Reply:
This works great, @indigo423!